libraries/SITL/SIM_Precland.h

```diff
@@ -38,13 +38,12 @@ class SIM_Precland {
 
     const Vector3d &get_target_position() const { return _target_pos; }
     bool is_enabled() const {return static_cast<bool>(_enable);}
-    void set_default_location(float lat, float lon, int16_t yaw);
     static const struct AP_Param::GroupInfo var_info[];
 
     AP_Int8 _enable;
-    AP_Float _origin_lat;
-    AP_Float _origin_lon;
-    AP_Float _origin_height;
+    AP_Float _precland_lat;
```

Comment:
I definitely like the rename of the parameter because "origin" is very easy to confuse with EKF origin.